DashSync/shared/Models/CoinJoin/Utils/DSMasternodeGroup.m

```diff
@@ -144,7 +179,73 @@ - (void)triggerConnections {
             return;
         }
         
-        // TODO
+        BOOL doDiscovery = NO;
+        NSDate *now = [NSDate date];
+        
+        @synchronized (self.inactives) {
```

Comment:
Looks like no reason to split sync into 2 parts here (it's expensive).

Reply:
fixed